### PR TITLE
CLOUD-2477 updates for consolidated modules

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHDM-XYZ] Subject` or `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: os.kieserver.s2i/module.yaml [hunk 1]
```diff
@@ -2,5 +2,8 @@ schema_version: 1
 name: os.kieserver.s2i
 version: '1.0'
 description: Legacy os.kieserver.s2i script package.
+envs:
+- name: MAVEN_LOCAL_REPO
```
Comment (errantepiphany) [inline os.kieserver.s2i/module.yaml]: @rcernich Should this env also be added to our 7.1 kieserver image?
This is the associated module in 7.1 that contains the s2i scripts:
https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/module.yaml

Comment (rcernich) [reply os.kieserver.s2i/module.yaml]: You'll probably need this once CLOUD-2477 is merged.  The default repo is now /tmp/artifacts/m2, which aligns better with save-artifacts and incremental builds, as we don't have to copy things around in the image.

**Comment by rcernich** [comment]
@errantepiphany, @spolti, please merge this.  The related changes in cct_module have been merged.  This needs to be merged at the same time as jboss-container-images/jboss-kieserver-6-openshift-image#48.

**Comment by errantepiphany** [comment]
It looks like this PR is safe to merge, since it only affects os.kieserver.*  modules. If it touched jboss-kie-* modules, then we wouldn't be able to merge it until after 7.1 is released, at which point will move from CE sprint-23 to CE sprint-24. @spolti , do you agree?

**Pull request merged**